fastparquet/writer.py

```diff
@@ -494,11 +504,10 @@ def make_metadata(data, has_nulls=[]):
 def write(filename, data, partitions=[0], encoding="PLAIN",
           compression=None, file_scheme='simple', open_with=default_openw,
           mkdirs=default_mkdirs, has_nulls=[]):
-    """ data is a 1d int array for starters
+    """ Write Pandas DataFrame to filename as Parquet Format
```

Comment:
Thanks for cleaning things up. I'll write comprehensive docs by the end of the week and flesh out some doc-strings along the way.